libraries/index.html

```diff
@@ -0,0 +1,121 @@
+title: Libraries
+permalink: /libraries
+---
+
+<div id="libraries-page" class="common-layout">
+  <div class="content">
+    <h1>CircuitPython Libraries</h1>
+    <p>
+      The CircuitPython Library Bundle contains all current libraries available for
+      CircuitPython. They are designed for use with CircuitPython and may or may not
+      work with MicroPython. The bundle options are explained below.
+    <p>
+      CircuitPython libraries are separate files designed to work with CircuitPython code.
+      CircuitPython programs require a lot of information to run. CircuitPython is so simple
+      to use because most of this information is processed in the background and stored in
+      libraries. Some libraries are built into CircuitPython. Others are downloaded and
+      stored on your CIRCUITPY drive in a folder called lib.
+    </p>
+    <p>
+      The full list of all the CircuitPython libraries contained in the following bundles
+      can be found on the
+      <a href="https://circuitpython.readthedocs.io/projects/bundle/en/latest/drivers.html" title="">libraries page</a>.
+    <h2>Download and Installation Instructions</h2>
+    <p>
+      The libraries in each release are compiled to .mpy for all recent major versions
+      of CircuitPython. Please download the bundle that matches the major version
+      of your CircuitPython. For example, if you are running 4.0.0 you should
+      download the 4.x bundle. The .mpy files are designed to take up less space on
+      your CIRCUITPY drive. Unless you have a specific reason, choose the appropiate
+      .mpy library bundle!
+    </p>
+    <p>
+      <b>To install</b>, download the appropriate bundle for your version of CircuitPython.
+      Unzip the file, open the resulting folder and find the lib folder. Open the lib
+      folder and find the library files you need to load. Create a lib folder on your
+      CIRCUITPY drive. Copy the individual library files you need to the lib folder
+      on your CIRCUITPY drive.
+    </p>
+    <p>
+      You can always find the
+      <a href="https://github.com/adafruit/Adafruit_CircuitPython_Bundle/releases/latest">latest releases of the libraries bundle</a>
+      on GitHub.
+    <div class="release-section">
+      <div id="adafruit-circuitpython-bundle-4.x-mpy">
+        <h3>Bundle Version 4.x</h3>
+        <p>
+          This bundle is built for use with CircuitPython 4.x.x. If you are using
+          CircuitPython 4, please download this bundle.
+        </p>
+      </div>
+      <div id="adafruit-circuitpython-bundle-3.x-mpy">
+        <h3>Bundle Version 3.x</h3>
+        <p>
+          This bundle is built for use with CircuitPython 3.x.x. If you are using
+          CircuitPython 3, please download this bundle.
+        </p>
+      </div>
+      <div id="adafruit-circuitpython-bundle-py">
+        <h3>Bundle Version py</h3>
+        <p>
+          This bundle is the uncompiled source code for every library. This bundle is
```

Comment:
Please change the text here to the following:
```
This bundle is the uncompiled source code for every library. It is not
intended for general use! It is only recommended if you need to edit
a library file. This bundle works with all versions of CircuitPython.
```
It is currently very redundant. (Which is obviously on me. I didn't catch it until last night.)

Reply:
Completed and pushed.